lib/scraper.rb

```diff
@@ -79,6 +79,8 @@ def run
         end
       end
 
+      scraper_event_check(data_sources)
```

Comment:
Maybe just call this from within the `scrape` method? Then you don't have to iterate over all the data sources again.

Reply:
I opted for afterwards since you can have multiple sources with the same content_provider so if you check right after the source specific scrape you are checking on events that have not been rescraped yet. Should not be a problem if the difference between your stale threshold and your scraping frequency is big enough but this felt cleaner.